colab/accounts/templates/accounts/manage_subscriptions.html

```diff
@@ -1,42 +1,35 @@
 {% extends 'base.html' %}
 {% load i18n gravatar %}
-
+{% load staticfiles %}
 {% block main-content %}
 
   <h2>{% blocktrans %}Group Subscriptions{% endblocktrans %}</h2>
   <h3>{% gravatar user_.email 50 %} {{ user_.get_full_name }} ({{ user_.username }})</h3>
   <br>
 
-  <form method='post'>
-    {% csrf_token %}
-
-    <div class="row">
-      {% for email, lists in membership.items %}
-      <div class="col-lg-6 col-md-6 col-sm-6 col-xs-12">
-        <div class="panel panel-default">
-          <div class="panel-heading">
-            <h3 class="panel-title">{{ email }}</h3>
-          </div>
-          <div class="panel-body">
-            {% for list, checked in lists %}
-              <div class="checkbox" title="{{ list.description }}">
-                <label>
-                  <input name="{{ email }}" value="{{ list.listname }}" type="checkbox" {% if checked %}checked{% endif%}>{{ list.listname }}</input>
-                </label>
-              </div>
-            {% endfor %}
-          </div>
-        </div>
+  {% block head_js %}
```

Comment:
@italopaiva , just one more subtle thing: I didn't know that, but apparently, when you include a 'block' within another 'block' tag ({block ..}{endblock}), django adds the generated content twice. So I noticed that for each character I typed on listname search, it was making 4 requests! This 'block_js' was moved to the top of the template and solved the problem =D: https://github.com/colab/colab/blob/detach_super_archives/colab/accounts/templates/accounts/manage_subscriptions.html#L5